nuxeo-webdav-test/src/test/java/org/nuxeo/ecm/webdav/WebDavClientTest.java

```diff
@@ -202,6 +202,33 @@ public void testListFolderContents() throws Exception {
         }
     }
 
+    @Test
+    public void testListFolderContentsSpecialName() throws Exception {
+        String folderName = "foo bar[1] caf\u00e9";
+        DocumentModel folder = session.createDocumentModel("/workspaces/workspace", folderName, "Folder");
+        session.createDocument(folder);
+        session.save();
+        TransactionHelper.commitOrRollbackTransaction();
+        TransactionHelper.startTransaction();
+
+        HttpPropfind request = new HttpPropfind(ROOT_URI, DavConstants.PROPFIND_ALL_PROP, DavConstants.DEPTH_1);
+        try (CloseableHttpResponse response = client.execute(request, context)) {
+            MultiStatus multiStatus = request.getResponseBodyAsMultiStatus(response);
+            MultiStatusResponse[] responses = multiStatus.getResponses();
+            StringBuilder failmsg = new StringBuilder("Got: ");
+            boolean gotit = false;
```

Comment:
Is it a failure?

Reply:
The `gotit` flag gets set to `true` if we find what we expect in the output. If there's no failure the `failmsg` is not used.